File: 30 Days of Code/Python/Day 25 - Running Time and Complexity.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
import math
t = int(input())
for i in range(t):
    n = int(input())
    f = 0
    if n==1:
        f=1
    else:
        for j in range(2,int(math.sqrt(n))+1):
            if n%j==0:
                f=1
                break
    if f==0:
        print("Prime")
    else:
        print("Not prime")

File: 30 Days of Code/PHP/Day 25 - Running Time and Complexity.php
<?php
$_fp = fopen("php://stdin", "r");
/* Enter your code here. Read input from STDIN. Print output to STDOUT */
$t = (int)readline();
for($i=0; $i<$t; $i++){
    $n = (int)readline();
    $f = 0;
    if ($n==1)
        $f=1;
    else{
        for($j=2; $j<((int)sqrt($n)+1); $j++){
            if($n%$j==0){
                $f=1;
                break;
        }
        }
    }
    if($f==0)
        echo("Prime"."\n");
    else
        echo("Not prime"."\n");
}
?>